web/fluidity.money/src/screens/MailingList/MailingList.tsx

```diff
@@ -17,9 +24,26 @@ const MailingList = () => {
   };
 
   const control = useAnimation();
-  
+  const [state, setState] =
```

Comment:
wouldn't this be better if it was just `state: "success" | "loading" | "failed"`, then you can switch on a single value when rendering